backend/app/tests/api/routes/test_onboarding.py

```
@@ -158,3 +172,203 @@ def test_onboard_project_with_auto_generated_defaults(
     assert "@kaapi.org" in data["user_email"]
     assert "api_key" in data
     assert len(data["api_key"]) > 0
+
+
+def test_onboard_project_duplicate_project_in_organization(
```

Ruff (0.14.5)

Redefinition of unused `test_onboard_project_duplicate_project_in_organization` from line 105

(F811)